mono/mini/mini-exceptions.c

```diff
@@ -121,6 +192,9 @@ mono_exceptions_init (void)
 	cbs.mono_exception_walk_trace = mono_exception_walk_trace;
 	cbs.mono_install_handler_block_guard = mono_install_handler_block_guard;
 	mono_install_eh_callbacks (&cbs);
+
+	mono_install_above_abort_threshold (mini_above_abort_threshold);
```

Comment:
It might be better to add a new function like mini_get_undeniable_exception () which would do this so there is no need to add callbacks.
It could also receive the sp as an argument so there is no need to do a stack walk.